09_iterators.md

```diff
@@ -404,11 +403,11 @@ Equality is naturally extendable to struct.
 It was an oversight on the part of C and still C++ again.
 The compiler should generate equality,
 and it could generate inequality using
-[lexographical][lex] ordering.
+[lexicoographical][lex] ordering.
```

Comment:
lexicographical. Looks like you have two oo's.

Reply:
Oops! Will fix!